css/style.css

```diff
@@ -0,0 +1,10 @@
+#nav-bar{
+    position: fixed;
+    text-align: left;
+    top: 0;
+    width:100;
+    display:flex;
```

Comment:
It is recommended to only use a single positioning technique per container. Either use Absolute positioning or flexbox!